src/main/kotlin/no/fdk/dataservicecatalog/controller/DataServiceController.kt

```diff
@@ -0,0 +1,72 @@
+package no.fdk.dataservicecatalog.controller
+
+import jakarta.validation.Valid
+import no.fdk.dataservicecatalog.domain.FindDataServiceQuery
+import no.fdk.dataservicecatalog.domain.JsonPatchOperation
+import no.fdk.dataservicecatalog.domain.RegisterDataServiceCommand
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ProblemDetail
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.MethodArgumentNotValidException
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/catalogs")
```

Comment:
JSON-endepunktene er kun for intern trafikk knyttet til frontend-skjemaet, det skal prefikses med /internal
```suggestion
@RequestMapping("/internal/catalogs")
```